testapi.pm

```diff
@@ -971,11 +973,12 @@ sub assert_script_run {
             # not change default timeout.
             timeout      => 90,
             fail_message => '',
-            quiet        => testapi::get_var('_QUIET_SCRIPT_CALLS')
+            quiet        => testapi::get_var('_QUIET_SCRIPT_CALLS'),
+            max_interval => undef
         }, ['timeout', 'fail_message'], @_);
 
     bmwqemu::log_call(cmd => $cmd, %args);
-    my $ret = $distri->script_run($cmd, timeout => $args{timeout}, quiet => $args{quiet});
+    my $ret = $distri->script_run($cmd, timeout => $args{timeout}, quiet => $args{quiet}, max_interval => $args{max_interval});
```

Comment:
Can you please try if it also works to just pass on all additional parameters? Then you would not even need to define line L977 and also do not need to mention `max_interval` explicitly here. Similar as we do in the next line `_handle_script_run_ret($ret, $cmd, %args);`

Reply:
I am pretty sure this would work, but do we really want to pass everything unattended?
Seems a bit risky to me

Reply:
I don't see a problem with that as we only use selected parameters.